### DISCUSSION TITLE
Dúvida de Uso / Usage Question

### DISCUSSION THREAD

**Discussion by iClaudioSantos** [skia4delphi/skia4delphi#311] [Q&A] [answered]
Estou usando o Skia4Delphi desde o Delphi 11.3.

Agora que saiu atualização para o Skia4Delphi, atualizarei a versão instalada no Delphi 11.3.

Porém, a dúvida é quanto ao Delphi 12: Tenho licença, então estava também utilizando em alguns projetos o Delphi 12, e quando lançou o Skia 6, foi junto com o lançamento do Delphi 12, até onde me consta é a mesma versão do Delphi 12 o Skia 6.

Agora lançou o Delphi 12.1 e o Skia 6.1

Eu devo instalar o Skia 6.1 no Delphi 12.1 ou já está incluso nele a versão?

//- - - - - - - - - - -

I've been using Skia4Delphi since Delphi 11.3.

Now that the update for Skia4Delphi is out, I will update the version installed in Delphi 11.3.

However, the question is about Delphi 12: I have a license, so I was also using Delphi 12 in some projects, and when Skia 6 was released, it was together with the release of Delphi 12, as far as I know it is the same version of Delphi 12 o Skia 6.

Now released Delphi 12.1 and Skia 6.1

Should I install Skia 6.1 in Delphi 12.1 or is the version already included?

**Comment by viniciusfbb** [comment, marked as answer]
Hello @iClaudioSantos

If you are on RAD 12.1, you are already updated:

- **RAD Studio 12.0 Athens:** Skia4Delphi v6.0 + [extra exclusive features](https://docwiki.embarcadero.com/RADStudio/Athens/en/Skia4Delphi_Exclusive_Features)
- **RAD Studio 12.1 Athens:** Skia4Delphi v6.1 + [extra exclusive features](https://docwiki.embarcadero.com/RADStudio/Athens/en/Skia4Delphi_Exclusive_Features)
